tests/integration/tier0/non-destructive/custom-repository/test_custom_repository.py

```diff
@@ -31,6 +31,12 @@ def test_custom_valid_repo_without_rhsm(convert2rhel, set_custom_repository_file
     enable_repo_opt = " ".join(f"--enablerepo {repo}" for repo in get_custom_repos_names())
 
     with convert2rhel(f"-y --no-rhsm {enable_repo_opt} --debug", unregister=True) as c2r:
+        c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+
+        # c2r.expect("Continue with the system conversion?")
+        c2r.sendline("y")
+
```

Comment:
Please remove this as well.

Reply:
gotcha I will remove these lines as well!